index.html

```diff
@@ -0,0 +1,268 @@
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h2>Profesional Art Printing Data</h2>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h2>Profesional Art Printing Data</h2>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h2>Profesional Art Printing Data</h2>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h2>Profesional Art Printing Data</h2>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+        </div>
+
+   </section>
+
+   <!--About Me Section-->
+
+   <section class="about-me">
+       <div class="my-details">
+          <h1>About me</h1>
```

Comment:
As the previous reviewer suggested, please update this element to use another `<hx>` tag such as `<h2>` to improve the semantics of your webpage🙏

Reply:
Fixed.